pennylane/circuit_graph.py

```diff
@@ -117,15 +117,15 @@ def __init__(self, ops, obs, wires, par_info=None, trainable_params=None):
             op.queue_idx = k  # store the queue index in the Operator
 
             if hasattr(op, "return_type"):
-                if op.return_type is qml.measurements.State:
+                if op.return_type is State:
                     # State measurements contain no wires by default, but wires are
                     # required for the circuit drawer, so we recreate the state
                     # measurement with all wires
-                    op = qml.measurements.MeasurementProcess(qml.measurements.State, wires=wires)
+                    op = MeasurementProcess(State, wires=wires)
 
-                elif op.return_type is qml.measurements.Sample and op.wires == Wires([]):
+                elif op.return_type is Sample and op.wires == Wires([]):
                     # Sampling without specifying wires is treated as sampling all wires
-                    op = qml.measurements.MeasurementProcess(qml.measurements.Sample, wires=wires)
+                    op = qml.sample(wires=wires)
```

Comment:
Would using this syntax have implicit queuing?

Reply:
I don't think so. `qml.sample(wires=wires)` is simply calling `MeasurementProcess(Sample, wires=wires) internally